### DISCUSSION TITLE
I cannot access my VPS due to firewall iP block

### DISCUSSION THREAD

**Discussion by eksqtr** [ovh/manager#5046] [Q&A] [answered]
I open a ticket but then no reply since then , I really need help with my VPS Server thing!

**Comment by antleblanc** [comment, marked as answer]
Hi @eksqtr!

Our support team will come back to you! Please refer to the ticket in the Control Panel.

From here (this repository/discussions' feed) we don't provide any support attached to your services.

Hope it helps you,
Thank!
